tests/unicode/unicode_write_count.py

```diff
@@ -0,0 +1,7 @@
+import sys
+
+n_text = sys.stdout.write("🚀\n")
+sys.stdout.write("{}\n".format(n_text))
```

Comment:
Wouldn't this be simpler?

```suggestion
print(n_text)
```

Reply:
I wanted the tests to cover both buffered and unbuffered writes for comparison.  With the other PR pulled in, we can have both together.